### ISSUE TITLE
Remove passing `normalize` argument to `sklearn.linear_model._base._pre_fit`

### ISSUE THREAD

**Issue by tobias-zehnder** [BoevaLab/sparsesurv#27]
Hello, thanks for this package, great work!

I would like to suggest to remove the normalize argument here. It has been deprecated in sklearn 1.2 and removed in 1.4.
https://github.com/BoevaLab/sparsesurv/blob/90b529222dc409c639989e9934cb2cf8fd63d768/sparsesurv/utils.py#L312

Thank you and best,
Tobias
